ci-operator/config/calebevans/windup_integration_test/README.md

```diff
@@ -0,0 +1,90 @@
+# windup-windup_integration_test-main<!-- omit from toc -->
+
+## Table of Contents<!-- omit from toc -->
+- [General Information](#general-information)
+- [Purpose](#purpose)
+- [Process](#process)
+  - [Cluster Provisioning and Deprovisioning: `cucushift-installer-rehearse-aws-ipi`](#cluster-provisioning-and-deprovisioning-cucushift-installer-rehearse-aws-ipi)
+    - [Provisioning](#provisioning)
+    - [Deprovisioning](#deprovisioning)
+  - [Orchestrate, Execute, and Report - `mtr-scenario`](#orchestrate-execute-and-report---mtr-scenario)
+- [Custom Images](#custom-images)
+  - [`mtr-runner`](#mtr-runner)
+
+
+## General Information
+
+- **Repository**: [windup/windup_integration_test](https://github.com/windup/windup_integration_test.git)
+- **Operator Tested**: [MTR (Migration Toolkit for Runtimes)](https://developers.redhat.com/products/mtr/overview)
+- **Maintainers**: Interop QE
+
+## Purpose
+
+To provision the necessary infrastructure and using that infrastructure to execute MTR interop tests. The results of theses tests should be reported to the appropriate sources following execution.
+
+## Process
+
+The execution of this configuration can be thought of as a 3 step process (with many sub-processes in each step):
+1. Provision the test cluster
+2. Orchestrate, Execute, and Report
+3. Deprovision the test cluster
+
+These steps use a combination of workflows, chains, and refs to complete the interop testing of the MTR operator. This documentation will reference the `tests:` stanza of the configuration file quite a bit, so here is the stanza for convenience:
+
+```yaml
+tests:
+- as: mtr-scenario
+  steps:
+    cluster_profile: aws-cspi-qe
+    env:
+      BASE_DOMAIN: aws.interop.ccitredhat.com
+      SELENIUM_NAMESPACE: mtr
+      SUB_INSTALL_NAMESPACE: mtr
+      SUB_PACKAGE: mtr-operator
+      SUB_SOURCE: redhat-operators
+      SUB_TARGET_NAMESPACES: mtr
+    test:
+    - chain: interop-mtr
+    workflow: cucushift-installer-rehearse-aws-ipi
+```
+
+### Cluster Provisioning and Deprovisioning: `cucushift-installer-rehearse-aws-ipi`
```

Comment:
Lines 51-72 roughly will be the same in each scenario that we create.

We could abstract this to a higher level piece of documentation then just add a reference to it here. 

I'm leaning towards adding the level of info directly to the workflows README (which doesn't exist right now)
https://github.com/openshift/release/tree/9a70897e1c01c7650df0e1b3dc188092cd600221/ci-operator/step-registry/cucushift/installer/rehearse/aws/ipi

Reply:
Or we could just direct people to the workflow doc that gets auto-created https://steps.ci.openshift.org/workflow/cucushift-installer-rehearse-aws-ipi

Reply:
Agreed - we can try to write our own documentation or reach out to the owners of that Workflow for some documentation.

Reply:
Updated in this commit: [9b549dc8ef74800853783297065b74950afe45f0](https://github.com/openshift/release/pull/34094/commits/9b549dc8ef74800853783297065b74950afe45f0)